doc/dev/release.md

```diff
@@ -45,6 +45,24 @@ $ git config [--global] user.signingkey "$GPG_KEY_ID"
 $ git config [--global] user.email "$GPG_EMAIL"
 ```
 
+Also, make sure that you setup the git gpg config as follows.
+```bash
+ cat ~/.gnupg/gpg.conf
+default-key $GPG_KEY_ID
```

Comment:
Is `default-key` a macOS command?

Reply:
No. It is required in any system and is checked in the Makefile. See; https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L135

Reply:
Ah I see why you have this in here.